newIDE/app/src/EffectsList/index.js

```diff
@@ -144,118 +204,197 @@ export default function EffectsList(props: Props) {
-                        <SelectField
-                          margin="none"
-                          value={effectType}
-                          onChange={(e, i, newEffectType: string) =>
-                            chooseEffectType(effect, newEffectType)
-                          }
-                          fullWidth
-                          hintText={t`Choose the effect to apply`}
                         >
-                          {allEffectMetadata.map(effectMetadata => (
-                            <SelectOption
-                              key={effectMetadata.type}
-                              value={effectMetadata.type}
-                              primaryText={effectMetadata.fullName}
-                              disabled={
-                                props.target === 'object' &&
-                                effectMetadata.isMarkedAsNotWorkingForObjects
-                              }
-                            />
-                          ))}
-                        </SelectField>
-                        <ElementWithMenu
-                          element={
-                            <IconButton>
-                              <MoreVert />
-                            </IconButton>
-                          }
-                          buildMenuTemplate={(i18n: I18nType) => [
-                            {
-                              label: i18n._(t`Delete`),
-                              click: () => removeEffect(effect.getName()),
-                            },
-                            { type: 'separator' },
-                            {
-                              type: 'checkbox',
-                              label: i18n._(t`Show Parameter Names`),
-                              checked: showEffectParameterNames,
-                              click: () =>
-                                setShowEffectParameterNames(
-                                  !showEffectParameterNames
-                                ),
-                            },
-                          ]}
-                        />
-                      </MiniToolbar>
-                      <Line expand noMargin>
-                        <Column expand>
-                          {!!effectType && effectMetadata ? (
-                            <React.Fragment>
-                              <Line>
-                                <BackgroundText>
-                                  <MarkdownText
-                                    source={effectMetadata.description}
+                          {({
+                            connectDragSource,
+                            connectDropTarget,
+                            isOver,
+                            canDrop,
+                          }) =>
+                            connectDropTarget(
+                              <div
+                                key={effect.ptr}
+                                style={{
```

Comment:
seeing inline style makes me wonder if we're doing something wrong

Reply:
You cannot give a React component to the connect functions of the drag n drop, they require pure HTML component :/

Reply:
👍